### PR TITLE
Update audio media injection in tinymce

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/45
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/126

### CODE REVIEW

File: MediaAdminBundle/Resources/views/BackOffice/Underscore/TinyMce/audio._tpl.twig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<img
+    class="tinymce-audio"
+    src="{% image '@OpenOrchestraMediaAdminBundle/Resources/public/images/icon/tinymce-audio.png' %}{{ asset_url }}{% endimage %}"
+    data-id="<%= media_id %>"
+/>
```
Comment (alavieille) [inline MediaAdminBundle/Resources/views/BackOffice/Underscore/TinyMce/audio._tpl.twig:5]: eof

File: MediaAdminBundle/Resources/views/BackOffice/Underscore/TinyMce/media._tpl.twig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<img
+    class="tinymce-media"
+    src="<%= media_src %>"
+    data-id="<%= media_id %>"
+    <% if (media_format) { %>data-format="<%= media_format %>"<% } %>
+/>
```
Comment (alavieille) [inline MediaAdminBundle/Resources/views/BackOffice/Underscore/TinyMce/media._tpl.twig:6]: eof

**Comment by alavieille** [comment]
else eof  :+1:

**Comment by ngilain** [comment]
@itkg-alavieille : updated

**Comment by alavieille** [comment]
:+1:

**Pull request merged**